GrayBMP.cs

```diff
@@ -126,6 +125,39 @@ public void Dirty ()
       End ();
    }
 
+   /// <summary>
+   /// Draws a thick line between the two given end-points (with given size & given shade of gray)
+   /// </summary>
+

```

Comment:
Style guide lines: No new line between comment and method. No new line for start and end tag in summary comments.

Reply:
Okay.